src/components/Calender/DateTile.tsx

```diff
@@ -64,21 +52,31 @@ export const DateTile = React.memo((props: DateTileProps) => {
             </TouchableOpacity>
         </Layout>
     );
-});
+}, areEqual);
+
+function areEqual(prevProps, nextProps): boolean {
+    if (prevProps.isSelected === nextProps.isSelected) return true; //no rerender
```

Comment:
could also code golf this to 
`return prevProps.isSelected === nextProps.isSelected`